babel/zh.ini

```diff
@@ -0,0 +1,216 @@
+[meta]
+name = 中文（繁體）
+language = zh
+contributors = 女神_a#3330
+inherit = 
+
+[main]
+command_reload_help = {p:local}{cmd} (extension)
+	在不重新啟動機器人的情況下重新加載給定的擴展。您也可以重新加載 config 或 babel 。
+extensions_list = 這裡是所有當前啟用的擴展
+	```{list}```
+extension_reload_success = 重新加載 {extension} 成功！
+extension_file_missing = 這個擴展的源文件丟失了！
+extension_not_found = 無法匹配您提供的擴展名與啟用的擴展名。
+
+[language]
+command_language_help = {p:local}{cmd} (get|set (langcode)|list)
+	更改與您或您的伺服器交談時將使用的語言。使用 `{p:local}language list` 查找翻譯，然後設置一個。
+	只有管理員可以更改伺服器語言。
+list_title = 可用語言
+set_howto = 使用 `{p:local}language set` 和下面的給定語言代碼設置語言。
+contribute_cta = 在這里為翻譯工作做出貢獻： {c:language/contribute_url}
+unknown_contributors = 沒有貢獻者
+coverage_label = *覆蓋率：* {coverage}%%
+origin_reason_author = 為您設置的語言
+origin_reason_guild = 為這個伺服器設置的語言
+origin_reason_default = 使用此機器人的默認語言
+set_failed_invalid_pattern = 您的語言代碼必須符合模式。（即 `en` 或 `br-PT` ，這是區分大小寫的）
+set_warning_no_match = 這種語言還沒有被翻譯，但如果有的話，你將是第一個看到它的人。
+set_success = 成功！ {c:main/botname} 現在會說 {language} {usermode?to you|in this guild}。
```

Comment:
I completely understand how this new system is a bit more confusing. So when you see a format like `{condition?true:false}`, you translate the `true` and `false` parts. In this case, for example, if `usermode` is true, the end string would be "ConfessionBot will now speak English (US) to you", if it's false, it would be "ConfessionBot will now speak English (US) in this guild".

You can also move the `{condition?true|false}` structure to any part of the sentence to make it gramatically correct.

Reply:
OK, I try it.
I only saw this message now because there was no notification lol

Reply:
The related text of {?|} has been modified, please check carefully for errors, all.